packages/backend-defaults/src/entrypoints/httpRouter/http/createLifecycleMiddleware.ts

```diff
@@ -25,15 +29,20 @@ export const DEFAULT_TIMEOUT = { seconds: 5 };
  * Options for {@link createLifecycleMiddleware}.
  * @public
  */
-export interface LifecycleMiddlewareOptions {
-  lifecycle: LifecycleService;
-  /**
-   * The maximum time that paused requests will wait for the service to start, before returning an error.
-   *
-   * Defaults to 5 seconds.
-   */
-  startupRequestPauseTimeout?: HumanDuration;
-}
+export type LifecycleMiddlewareOptions =
+  | {
+      lifecycle: LifecycleService;
+      /**
+       * The maximum time that paused requests will wait for the service to start, before returning an error.
+       *
+       * Defaults to 5 seconds.
+       */
+      startupRequestPauseTimeout?: HumanDuration;
```

Comment:
ah but I think we should remove this completely :sweat_smile:, there's not that much use to being able to set this via code is there?

Reply:
It is a breaking change, right 🤔 ?